### PR TITLE
sidebar - active component styling and dropdown effect

### PR DESCRIPTION
**Pull request by odedre**
## Proposed Changes

- sidebar active option ui
- sidebar dropdown effect
- separated component node to its own folder

### CODE REVIEW

File: src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
 import classNames from 'classnames';
-import { NavLink } from 'react-router-dom';
 import { Icon } from '@bit/bit.evangelist.elements.icon';
 
 import { TreeNodeProps, TreeLayer } from '../recursive-tree';
-import { ComponentTreeContext } from '../component-tree-context';
 import { indentStyle, indentClass } from '../indent';
 import { getName } from '../utils/get-name';
 import { clickable } from '../../../../../../to-eject/css-components/clickable';
-import { hoverable } from '../../../../../../to-eject/css-components/hoverable';
```
Comment (ranm8) [inline src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx:9]: remove line space

File: src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx [hunk 2]
```diff
@@ -1,14 +1,11 @@
-import React, { Component, useState } from 'react';
+import React, { useState } from 'react';
 import classNames from 'classnames';
-import { NavLink } from 'react-router-dom';
 import { Icon } from '@bit/bit.evangelist.elements.icon';
```
Comment (ranm8) [inline src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx:4]: remove line space

File: src/extensions/workspace/ui/side-bar/component-tree/component-view/component-view.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+import { TreeNodeProps } from '../recursive-tree';
+import { ComponentTreeContext } from '../component-tree-context';
+import { indentClass } from '../indent';
+import { getName } from '../utils/get-name';
+import { clickable } from '../../../../../../to-eject/css-components/clickable';
+import { hoverable } from '../../../../../../to-eject/css-components/hoverable';
+
```
Comment (ranm8) [inline src/extensions/workspace/ui/side-bar/component-tree/component-view/component-view.tsx:11]: remove line space

File: src/extensions/workspace/ui/side-bar/component-tree/component-view/component-view.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
```
Comment (ranm8) [inline src/extensions/workspace/ui/side-bar/component-tree/component-view/component-view.tsx:4]: remove line space

Comment (odedre) [reply src/extensions/workspace/ui/side-bar/component-tree/component-view/component-view.tsx:4]: done

File: src/extensions/workspace/ui/side-bar/component-tree/component-view/component-view.tsx (hunk 3 above)
Comment (odedre) [reply src/extensions/workspace/ui/side-bar/component-tree/component-view/component-view.tsx:11]: done

File: src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx (hunk 2 above)
Comment (odedre) [reply src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx:4]: done

File: src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx (hunk 1 above)
Comment (odedre) [reply src/extensions/workspace/ui/side-bar/component-tree/component-nodes/component-nodes.tsx:9]: done

**Pull request merged**